frontend/src/views/StatblockEditorView.vue

```diff
@@ -457,6 +451,45 @@
 				</div>
 			</template>
 		</Modal>
+
+		<Modal :show="showCRModal" @close="showCRModal = false">
+			<template #header>Calculate Monster CR</template>
+			<template #body>
+				<span>CR Calculation is an approximate at best. User discretion is advised. Initial numbers are approximates based on the current monster statblock.</span>
+				<h2 class="group-header">Calculator Output</h2>
+				<div class="three-wide">
+					<LabelledComponent title="Calculated CR"><span>{{crCalc.totalCR}}</span></LabelledComponent>
+					<LabelledComponent title="Offensive CR"><span>{{crCalc.offensiveCR}}</span></LabelledComponent>
+					<LabelledComponent title="Defensive CR"><span>{{crCalc.defensiveCR}}</span></LabelledComponent>
+					<LabelledComponent title="Proficiency Bonus"><span>{{crCalc.proficiencyBonus}}</span></LabelledComponent>
+					<LabelledComponent title="XP"><span>{{crCalc.xp}}</span></LabelledComponent>
+				</div>
+
+				<hr/>
+
+				<h2 class="group-header">Parameters</h2>
+				<LabelledNumberInput v-model="crCalc.expectedCR" title="Expected CR" :steps="ChallengeRatingsList"></LabelledNumberInput>
+				<div class="two-wide editor-field__container">
+					<LabelledNumberInput v-model="crCalc.hp" title="Effective HP" :step="1" :min="0"></LabelledNumberInput>
+					<LabelledComponent title="Effective HP Calculation"><span>{{crCalc.hpString}}</span></LabelledComponent>
+					<LabelledNumberInput v-model="crCalc.ac" title="Effective AC" :step="1" :min="0"></LabelledNumberInput>
+					<LabelledComponent title="Effective AC Calculation"><span>{{crCalc.acString}}</span></LabelledComponent>
+					<LabelledNumberInput v-model="crCalc.dpr" title="Damage Per Round (DPR)" :step="1" :min="0"></LabelledNumberInput>
+					<LabelledComponent title="Can fly and deal damage at range?">
+						<span><input type="checkbox" v-model="crCalc.flies"/> CR [0-9] only. Adjusts the effective ac</span>
```

Comment:
Checkboxes should use a label. see [here](https://github.com/Bestiary-Builder/Bestiary-Builder/blob/b3dc3d839328fc679a27625dc32dab9f0438b3be/frontend/src/views/StatblockEditorView.vue#L53)